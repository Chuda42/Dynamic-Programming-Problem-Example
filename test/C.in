leer_datos
10 1 20 100 .
leer_rendimiento
100 3 2 1 .
max_datos_procesados
planificacion_optima
Fin
