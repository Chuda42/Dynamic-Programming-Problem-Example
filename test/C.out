1>2>3>Max Datos Procesados: 111
4>Cantidad Datos Planificacion: 111
5>Fin.
